extensions/amp-form/0.1/amp-form.js

```diff
@@ -99,6 +150,11 @@ export class AmpForm {
   /** @private */
   installSubmitHandler_() {
     this.form_.addEventListener('submit', e => this.handleSubmit_(e), true);
+    const inputs = this.form_.querySelectorAll('input,select,textarea');
```

Comment:
I definitely agree. I don't know if `input` bubbles and definitely `blur` doesn't. I vaguely remember there was a `focusout` event that did bubble, so maybe that would work.